session_0/word_processing.rb

```diff
@@ -18,7 +26,13 @@ def lower_case!(words)
 # words_with_prefix('apple', ['apple', 'ball', 'applesauce']) would
 # return the words 'apple' and 'applesauce'.
 def words_with_prefix(prefix, words)
-  raise NotImplementedError # TODO
+  array = Array.new
+  words.each do |word|
```

Comment:
Do you think select method can be of use here? 

https://www.rubyguides.com/2019/04/ruby-select-method/